docs/api.asciidoc

```diff
@@ -0,0 +1,159 @@
+ * `context`: A dictionary with contextual information. This dictionary must follow the 
+    https://www.elastic.co/guide/en/apm/server/current/error-api.html#error-context-schema[Context] schema definition.
+ * `custom`: A dictionary of custom data you want to attach to the event.
+
+Returns the id of the error as a string.
+
+[break]
+[[client-api-capture-exception]]
+==== `Client.capture_message()`
+
+Captures a message with optional added contextual data.
+
+[source,python]
+----
+client.capture_message('Billing process succeeded.')
+----
+
+[break]
+===== Arguments
+
+ * `message`: The message as a string.
+ * `param_message`: Alternatively, a parametrized message as a dictionary.
+    The dictionary contains two values: `message`, and `params`.
+    This allows the APM server to group messages together that have share the same
+    parametrized message. Example:
++
+[source,python]
+----
+client.capture_message(param_message={
+    'message': 'Billing process for %s succeeded. Amount: %s',
+    'params': (customer.id, order.total_amount),
+})
+----
++
+ * `stack`: If set to `True` (the default), a stacktrace from the call site will be captured.
+ * `exc_info`: A `(type, value, traceback)` tuple as returned by https://docs.python.org/3/library/sys.html#sys.exc_info[`sys.exc_info()`]. If not provided, it will be captured automatically, if `capture_message()` was called in an `except` block.
+ * `date`: A `datetime.datetime` object representing the occurrence time of the error. If left empty, it defaults to `datetime.datetime.utcnow()`.
+ * `context`: A dictionary with contextual information. This dictionary must follow the 
+    https://www.elastic.co/guide/en/apm/server/current/error-api.html#error-context-schema[Context] schema definition.
+ * `custom`: A dictionary of custom data you want to attach to the event.
+
+Returns the id of the message as a string.
+
+[[transaction-api]]
+=== Transaction API
+
+[break]
+[[client-api-begin-transaction]]
+==== `Client.begin_transaction()`
+Begin tracking a transaction.
+Should be called e.g. at the beginning of a request or when starting a background task.
+
+[break]
+===== Arguments
+
+ * `transaction_type`: A string describing the type of the transaction, e.g. `'request'` or `'celery'`.
+
+[break]
+[[client-api-end-transaction]]
+==== `Client.end_transaction()`
+End tracking the transaction.
+Should be called e.g. at the end of a request or when ending a background task.
+
+[break]
+===== Arguments
+
+ * `name`: A string describing the name of the transaction, e.g. `process_order`.
+   This is typically the name of the view/controller that handles the request, or the route name.
+ * `result`: A string describing the result of the transaction.
+   This is typicaly the HTTP status code, or e.g. `'success'` for a background task.
+
+[break]
+[[api-elasticapm-instrument]]
+==== `elasticapm.instrument()`
```

Comment:
This doesn't seem to be related to `transaction` specific cmds, so maybe you want to move it to its own section like `Auto-Instrumentation`.